datahub-web-react/src/app/lineage/LineageEntityColumns.tsx

```diff
@@ -60,7 +67,8 @@ export default function LineageEntityColumns({ node, onHover }: Props) {
     }, [displayedFields, node?.data?.urn, setVisibleColumnsByUrn, previousDisplayedFields]);
 
     const hasColumnPagination =
```

Comment:
minor: can we first create another variable called columns? 

columns = node.data.schemaMetadata.... || node.data.inputFields

Reply:
yup I can do this in my next PR! that'll be cleaner to read